### PR TITLE
[TS] LPS-171968 Error popup shows when loading a page with an image for which the user does not have view permission

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

Motivation
=======
When a user is viewing a web content with an image that he has no permission to view, a pop-up error message will appear, with the generic, default message "Your request failed to complete".
This issue appeared after [LPS-156404](https://issues.liferay.com/browse/LPS-156404).

As @dacousalr identified it, the root cause is that the new `StrutsAction` is rendering the _portal_normal_ template without calling `SessionErrors.clear(request)` afterward.

Proposed Solution
========
Clearing the request from session errors.

Steps to Verify
========
Please see the linked LPS for the reproduction steps.
https://issues.liferay.com/browse/LPS-171968

Thank you and best regards,
István

### CODE REVIEW

File: modules/apps/layout/layout-utility-page/layout-utility-page-status/src/main/java/com/liferay/layout/utility/page/status/internal/struts/StatusStrutsAction.java [hunk 1]
```diff
@@ -71,15 +74,22 @@ public String execute(
...
+		SessionErrors.clear(httpServletRequest);
+
 		Document document = Jsoup.parse(
 			ThemeUtil.include(
 				httpServletRequest.getServletContext(), httpServletRequest,
 				httpServletResponse, "portal_normal.ftl", layoutSet.getTheme(),
 				false));
 
+		PortalMessages.clear(httpServletRequest);
+		SessionMessages.clear(httpServletRequest);
```
Comment (jkappler) [inline modules/apps/layout/layout-utility-page/layout-utility-page-status/src/main/java/com/liferay/layout/utility/page/status/internal/struts/StatusStrutsAction.java:88]: Hi @IstvanD, thanks for the PR! I have a question, do we have to clear twice the session messages?

Thanks!

Comment (IstvanD) [reply modules/apps/layout/layout-utility-page/layout-utility-page-status/src/main/java/com/liferay/layout/utility/page/status/internal/struts/StatusStrutsAction.java:88]: Hi @jkappler the two statements are clearing two different sets of messages. @dacousalr please correct me if I'm wrong. Thank you!

Comment (jkappler) [reply modules/apps/layout/layout-utility-page/layout-utility-page-status/src/main/java/com/liferay/layout/utility/page/status/internal/struts/StatusStrutsAction.java:88]: perfect @IstvanD. Thanks to you and @dacousalr!

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**